packages/grid/src/elements/splitter/components/Content.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, HTMLAttributes } from 'react';
+import { StyledPaneItem } from '../../../styled/splitter/StyledPaneItem';
+
+const ContentComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  (props, ref) => {
+    return <StyledPaneItem ref={ref} {...props} />;
```

Comment:
Would it make more sense to name this `StyledPaneContent` to establish affinity with the element-level component?

Reply:
Fixed in your changes